opm/io/eclipse/ESmry.hpp

```diff
@@ -104,13 +104,13 @@ class ESmry
 
     time_point startdat;
 
-    std::vector<std::string> checkForMultipleResultFiles(const Opm::filesystem::path& rootN, bool formatted) const;
+    std::vector<std::string> checkForMultipleResultFiles(const filesystem::path& rootN, bool formatted) const;
 
     void getRstString(const std::vector<std::string>& restartArray,
-                      Opm::filesystem::path& pathRst,
-                      Opm::filesystem::path& rootN) const;
+                      filesystem::path& pathRst,
```

Comment:
What was the reason we could not use plain `std::filesystem` again? Lagging support in macos?

Reply:
gcc7 (ubuntu bionic).